src/app/app-routing.module.ts

```diff
@@ -5,13 +5,22 @@ import { DashboardComponent } from './components/dashboard/dashboard.component';
 import { ErrorComponent } from './components/error/error.component';
 import { LoginComponent } from './components/login/login.component';
 import { PasswordResetComponent } from './components/password-reset/password-reset.component';
+import { ProjectDashboardComponent } from './components/project-dashboard/project-dashboard.component';
 import { RegisterComponent } from './components/register/register.component';
 import { AuthGuard } from './utils/auth.guard';
-import { BoardGuard } from './utils/board.guard';
+import { ProjectBoardGuard } from './utils/project.board.guard';
+import {ProjectGuard} from './utils/project.guard'
 
 const routes: Routes = [
-  { path: 'canvas/:boardID', component: CanvasComponent,
-    canActivate: [BoardGuard] 
+  {
+    path:'project/:projectID',
+    component:ProjectDashboardComponent,
+    canActivate:[ProjectGuard]
+  },
+  { path: 'canvas/:projectID/:boardID',
```

Comment:
Is it possible to change this to something like `project/:projectID/board/:boardID` ?

Reply:
@Ammar-T updated routing as per your suggestion